Tailviewer.AcceptanceTests/BusinessLogic/LogFiles/TextLogFileTest.cs

```diff
@@ -126,16 +124,16 @@ public void TestDoesNotExist()
 		{
 			_file = new TextLogFile(_scheduler, _fname);
 			_scheduler.RunOnce();
-			_file.GetValue(LogFileProperties.EmptyReason).Should().Be(ErrorFlags.None);
-			_file.GetValue(LogFileProperties.Created).Should().NotBe(DateTime.MinValue);
+			_file.Error.Should().Be(ErrorFlags.None);
+			_file.Created.Should().NotBe(DateTime.MinValue);
 
 			_streamWriter?.Dispose();
 			_stream?.Dispose();
 			File.Delete(_fname);
 			_scheduler.RunOnce();
 
-			_file.GetValue(LogFileProperties.EmptyReason).Should().Be(ErrorFlags.SourceDoesNotExist);
```

Comment:
This merge doesn't look right, GetValue(...) is the new way to retrieve data, please don't revert that.